logstash/Dockerfile

```diff
@@ -49,11 +54,25 @@ ADD env2yaml/env2yaml /usr/local/bin/
 EXPOSE 9600 5044
 
 
-LABEL org.label-schema.schema-version="1.0" \
+LABEL  org.label-schema.schema-version="1.0" \
   org.label-schema.vendor="Elastic" \
+  org.opencontainers.image.vendor="Elastic" \
   org.label-schema.name="logstash" \
-  org.label-schema.version="7.9.3" \
+  org.opencontainers.image.title="logstash" \
+  org.label-schema.version="7.10.0" \
+  org.opencontainers.image.version="7.10.0" \
   org.label-schema.url="https://www.elastic.co/products/logstash" \
   org.label-schema.vcs-url="https://github.com/elastic/logstash" \
-license="Elastic License"
+  license="Elastic License" \
+  org.label-schema.license="Elastic License" \
+  org.opencontainers.image.licenses="Elastic License" \
+  org.label-schema.build-date=2020-11-10T00:17:56.1604967476Z \
+  org.opencontainers.image.created=2020-11-10T00:17:56.1604967476Z \
+  description="Logstash is a free and open server-side data processing pipeline that ingests data from a multitude of sources, transforms it, and then sends it to your favorite 'stash.'" \
+  name="logstash" \
+  maintainer="info@elastic.co" \
+  summary="Logstash is a free and open server-side data processing pipeline that ingests data from a multitude of sources, transforms it, and then sends it to your favorite 'stash.'" \
+  vendor="Elastic"
```

Comment:
(Copying my comments from https://github.com/elastic/dockerfiles/commit/a3f88dac1835d583141d98d2cbae889492b9d09b to hopefully get more visibility on them :innocent:)

There seems to be a minor difference here between the published images and the published source:

```diff
-LABEL ... org.label-schema.build-date=2020-11-10T00:13:15.1604967195Z org.opencontainers.image.created=2020-11-10T00:13:15.1604967195Z ...
+LABEL ... org.label-schema.build-date=2020-11-10T00:17:56.1604967476Z org.opencontainers.image.created=2020-11-10T00:17:56.1604967476Z ...
```

Was this delta intentional?

---

Additionally, these added non-reverse-DNS labels are [not OCI compliant](https://github.com/opencontainers/image-spec/blob/v1.0.1/annotations.md).

(Also, `org.label-schema` is officially deprecated for quite some time now, FWIW)

Reply:
@elastic/logstash can someone take a look at this?

Reply:
I'll take a look at the non-compliant freeform labelling and the date discrepancies and should be able to put a PR in the logstash docker build to avoid this for future builds

Reply:
Thanks @robbavey , and thanks @tianon for raising